### ISSUE TITLE
[npm] Can't install on Node 8.9.4 due to package.json specified <8.9.3

### ISSUE THREAD

**Issue by adamJLev** [indexeddbshim/IndexedDBShim#317]
```
$ node --version
v8.9.4

$ yarn add indexeddbshim
yarn add v1.4.0
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
[3/5] 🚚  Fetching packages...
error indexeddbshim@3.5.0: The engine "node" is incompatible with this module. Expected version ">=6.9.2 < 8.9.3 || >= 9.0.0".
error An unexpected error occurred: "Found incompatible module".
info If you think this is a bug, please open a bug report with the information provided in "/Users/web/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
(ve)

```

**Comment by brettz9** [comment]
I added the engines limitation in package.json (commit a94cd5f ) because 8.9.3 consistently had a problem with one of our Mocha tests "query multi-entry indexes with hundreds of records" (and I now see neither 8.9.4 nor 9.0.0 fixed the problem, but 9.1.0 did fix it, so I could at least allow it again starting with 9.1.0--you can run the test file that includes the problematic test locally with `npm --test=IDBIndex/openCursor-spec.js run mocha` to confirm). 

Maybe one test failure wasn't enough to justify the engines exclusion, but it did seem like a quirky enough problem that people might wish to notified of a special issue and upgrade. But if this is a real issue where people have to use that version, we could remove the engines limitation and just make a note of the issue in the docs, though it'd be less likely to be discovered.

**Comment by fkleuver** [comment]
I'm developing a plugin for aurelia and lokijs that primarily targets the browser and this failed to install for me (on a machine running node 8.10.0). Had to override it with `--ignore-engines`

I wish this were a clear warning rather than a failing error, or that the `--ignore-engines` option would somehow be included in the error. I can't have this failure trickle down to consumers of my plugin who probably don't need the shim in most cases, let alone will encounter this problem (because they're using it in the browser).

I certainly won't be upgrading my node version higher than the current LTS version, and I also won't be downgrading it due to other things being dependent on the latest LTS. I believe the same goes for many others. So for browser users this seems like quite the overkill.

**Comment by brettz9** [comment]
Ok, I've re-enabled all engines after 6.9.2 (I don't know how well these earlier versions actually perform, btw, but that's what we had). I just made a note to the README about the test failure as a known issue. Thanks for the feedback! (Pushed to `master` and published to npm as 3.5.1)
